src/main/java/io/github/togar2/pvp/feature/damage/VanillaDamageFeature.java

```diff
@@ -227,7 +228,7 @@ protected void handleDamage(EntityDamageEvent event) {
 		} else if (hurtSoundAndAnimation) {
 			// Workaround to have different types make a different sound,
 			// but only if the sound has not been changed by damage#getSound
-			if (entity instanceof Player && sound == SoundEvent.ENTITY_PLAYER_HURT) {
+			/*if (entity instanceof Player && sound == SoundEvent.ENTITY_PLAYER_HURT) {
```

Comment:
Why comment this out?

Reply:
Nevermind I see why. Probably should add the sound effect to the damage type info?

Reply:
Made a PR to reintroduce the effects field. https://github.com/Minestom/Minestom/pull/2793

Reply:
Thanks, I will update the pr once it's added. Forgot to mention it in the pr message, commented it out to test my pr on my test server

Reply:
Just to let you know, the PR has been merged

Reply:
added